docs/pages/access-controls/idps/saml-reference.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: SAML Identity Provider Reference
+description: Reference documentation for the SAML identity provider
+---
+
+This page provides details on the SAML identity provider built into Teleport.
+
+## What is the SAML identity provider?
+
+The SAML identity provider allows users to use Teleport to authenticate and
+authorize to external applications outside of Teleport that support SAML
+Single Sign-On.
+
+## Basic definitions
+
+The following definitions are paraphrased and summarized from OASIS's SAML
+specification documents, referenced at the bottom of this page.
+
+### Assertions
+
+A package of information that supplies zero or more statements made by an
+identity provider. Assertions are usually made about a subject, which in
+Teleport's case will be Teleport users. These can be viewed as statements of
+fact about a user, e.g. "user *example* belongs to roles *editor*, *access*."
+
+### Service Provider
+
+A service provider can make use of assertions about a subject in order to control
+access and provide customized service. They rely on the **identity provider** for
+these assertions. In Teleport's case, service providers are typically applications
+like Slack, Vercel, etc. that allow SAML Single Sign-On.
+
+### Identity Provider
+
+A service that makes assertions about a subject.
+
+### Service/identity provider initiated SSO
+
+The login process can be initiated by either the service provider or the entity
+provider. The processes to trigger either of these are subtly different. The
```

Comment:
I think that in our case the "entity provider" and "identity provider" are both the same thing, Teleport? If so, since this is the only place where we reference an "entity provider", should this just be "identity provider"?

Reply:
I don't see `entity provider` here. In this case, service providers and identity providers are both different.

Reply:
>The login process can be initiated by either the service provider or the **entity** provider.

Reply:
Ahhh whoops. That's supposed to be identity... sorry about that!